src/sections/para-que-serve/index.module.css

```diff
@@ -0,0 +1,40 @@
+@import "../../styles/all.module.css";
+
+.root {
+  max-width: 1220px;
+  padding: 74px 0;
+  margin: 0 auto;
+  text-align: center;
+
+
+  @include breakpoint-mobile-only {
+    padding: 48px 36px;
+  }
+}
+
+.subtitle {
+  max-width: 676px;
+  margin: 21px auto 0 auto;
+}
+
+.content {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
We also need to align items vertically to the center and `max-height: 317px` (only desktop).
```css
align-items: center;
```